packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -404,6 +406,14 @@ export class VaFileInput {
             />
             {uploadStatus === 'idle' && (
               <div>
+                <span id="file-input-error-alert" role="alert">
+                  {displayError && (
+                    <Fragment>
+                      <span class="usa-sr-only">{i18next.t('error')}</span>
+                      <span class="usa-error-message">{displayError}</span>
+                    </Fragment>
+                  )}
+                </span>
```

Comment:
Currently the file input didn't display an error message before a file was successfully uploaded. Added this message to show if there is an error before a file is uploaded.